packages/teams-js/src/public/runtime.ts

```diff
@@ -102,7 +102,7 @@ export const teamsRuntimeConfig: IRuntime = {
       update: {},
     },
     files: {},
-    logs: {},
+    log: {},
     media: {},
     meeting: {},
     meetingRoom: {},
```

Comment:
Note: I don't think the `versionConsts` on line 147 should be exported. Can you fix this while you're touching this file please?

Reply:
We need to export it as we are using it in `runtime.ts`

Reply:
This is in runtime.ts :) You can remove the export. I tested it and it works.

Reply:
It'll work but when doing unit test cases, we need this function so if there is any other way, please let me know

Reply:
Ahhh gotcha. Okay, fine to leave!